Lib/threading.py

```diff
@@ -895,6 +895,24 @@ class is implemented.
                 except AttributeError:
                     pass
 
+        # Only check when a program uses pyrona
+        if is_pyrona_program():
+            def movable(o):
+                return isinstance(o, Region) and not o.is_open()
+
+            for k, v in kwargs.items():
+                if not (isimmutable(v) or isinstance(v, Cown) or movable(v)):
+                    raise RuntimeError(f'thread was passed {k} : {type(v)} -- '
```

Comment:
It feels like this should be a specific error and not just a generic `RuntimeError`. Not a blocker for this PR, but could you add a TODO for it? Or if you think this should be the final error type I'd be interested in the reasoning :)

Reply:
I had similar thoughts. I'll add a TODO and move on!

Reply:
Fixed!